src/components/shopping-cart-drop-down/shopping-cart-drop-down.component.html

```diff
@@ -0,0 +1,27 @@
+<ul class="navbar-right">
+    <div class="dropdown">
+    <ul><button  class="btn btn-link dropdown-toggle" data-bs-toggle="dropdown" type="button" href="" id="cart">
+
+        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-cart" viewBox="0 0 16 16">
```

Comment:
We could try using bootstrap icons for these instead: https://icons.getbootstrap.com/

Reply:
what I used is a bootstrap icon me finks